pandas/tests/io/pytables/common.py

```diff
@@ -51,7 +45,7 @@ def ensure_clean_store(path, mode="a", complevel=None, complib=None, fletcher32=
     finally:
         safe_close(store)
         if mode == "w" or mode == "a":
-            safe_remove(path)
+            tm.ensure_clean(path)
```

Comment:
hmm you cannot do this here as its a context manager. (and its already within a context manger)

Reply:
instead we need to leave using `os.remove` (and exempt this file from the check i think)